File: ProyectoCsharp/Modelo/Usuario.cs
namespace Modelo
{
    public abstract class Usuario
    {
        private long id_usuario;
        private string nombre;
        private string apellidos;
        private string correo;
        private string dni;
        private string username;
        private string contraseņa;
        private string descripcion_permisos;


        //Constructores
        public Usuario()
        {
            id_usuario = 0;
            nombre = apellidos = correo = Dni = contraseņa = descripcion_permisos = Username = null;
        }

        public Usuario(long id_usuario, string nombre, string apellidos, string correo, string dni, string username,
                       string contraseņa, string descripcion_permisos)
        {
            Id_usuario = id_usuario;
            Nombre = nombre;
            Apellidos = apellidos;
            Correo = correo;
            Dni = dni;
            Contraseņa = contraseņa;
            Descripcion_permisos = descripcion_permisos;
        }

        //Destructor
        ~Usuario()
        {
            nombre = apellidos = correo = dni = contraseņa = descripcion_permisos = null;
        }

        public long Id_usuario { get => id_usuario; set => id_usuario = value; }
        public string Nombre { get => nombre; set => nombre = value; }
        public string Apellidos { get => apellidos; set => apellidos = value; }
        public string Correo { get => correo; set => correo = value; }
        public string Contraseņa { get => contraseņa; set => contraseņa = value; }
        public string Descripcion_permisos { get => descripcion_permisos; set => descripcion_permisos = value; }
        public string Username { get => username; set => username = value; }
        public string Dni { get => dni; set => dni = value; }

        public override string ToString()
        {
            return string.Format("{0} {1}", Nombre, Apellidos);
        }
    }
}

File: TrabajoJavaLp2/src/modelo/Usuario.java
package modelo;
public abstract class Usuario{
	private long id_usuario;
	private String nombre;
	private String apellidos;
	private String correo;
	private String codigo;
	private String contrasena;
	private String descripcion_permisos;

	public Usuario(){
		id_usuario = 0;
		nombre = apellidos = correo = codigo = contrasena = descripcion_permisos = null;
	}

	public Usuario(long id_usuario, String nombre, String apellidos, String correo, String codigo, String contrasena, String descripcion_permisos) {
            this.id_usuario = id_usuario;
            this.nombre = nombre;
            this.apellidos = apellidos;
            this.correo = correo;
            this.codigo = codigo;
            this.contrasena = contrasena;
            this.descripcion_permisos = descripcion_permisos;
        }

	public void setId_usuario(long id_usuario){
		this.id_usuario = id_usuario;
	} 

	public long getId_usuario(){
		return id_usuario;
	}

	public void setNombre(String nombre){
		this.nombre = nombre;
	}

	public String getNombre(){
		return nombre;
	}

	public void setApellidos(String apellidos){
		this.apellidos = apellidos;
	}

	public String getApellidos(){
		return apellidos;
	}

	public void setCorreo(String correo){
		this.correo = correo;
	}

	public String getCorreo(){
		return correo;
	}

	public void setCodigo(String codigo){
		this.codigo = codigo;
	}

	public String getCodigo(){
		return codigo;
	}

	public void setContrasena(String contrasena){
		this.contrasena = contrasena;
	}

	public String getContrasena(){
		return contrasena;
	}

	public void setDescripcion_permisos(String descripcion_permisos){
		this.descripcion_permisos = descripcion_permisos;
	}

	public String getDescripcion_permisos(){
		return descripcion_permisos;
	}
}